docs/install-on-microk8s.md

```diff
@@ -0,0 +1,49 @@
+# Install on Minikube
+
+## Pre-requisites
+- [MicroK8s](https://microk8s.io/)
+- To install etcd using these instructions, you must have [Helm](https://helm.sh) in your client environment.
+- [Download the Compose on Kubernetes installer](https://github.com/docker/compose-on-kubernetes/releases).
+- Ensure storage and dns are enabled: `microk8s.enable storage dns`
+- Alias kubectl from the snap with: `sudo snap alias microk8s.kubectl kubectl`.
+
+## Create compose namespace
+
+- Confirm that MicroK8s is working with `microk8s.inspect`.
+- Create a compose namespace by running `kubectl create namespace compose`.
+
+## Deploy etcd
+
+Please follow [How to deploy etcd](./deploy-etcd.md).
+
+## Deploy Compose on Kubernetes
+
+If you do not already have a `~/.kube/config` file create it with `kubectl config view --raw > ~/.kube/config`.

```

Comment:
in case the user has both kubectl and microk8s, it makes more sense to suggest:
```suggestion
If you do not already have a `~/.kube/config` file create it with `microk8s.kubectl config view --raw > ~/.kube/config`.
```

Reply:
Also note, it is required before actually deploying etcd

Reply:
I don't believe this is accurate, I can disable the config with `mv ~/.kube{,.disabled}` and both `helm` and `kubectl` work fine, which is all that's required for deploying etcd.

Reply:
Adjusted to use `microk8s.kubectl` and shifted the block above the etcd deploy in 75fa1d1 anyway.